cpp/common/env.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) ZeroC, Inc.
+
+#include <optional>
+#include <string>
+
+namespace common
+{
+    /// Get the value of an environment variable.
+    /// @param variableName The name of the environment variable.
+    /// @return The value of the environment variable, or nullopt if the variable is not set.
+    std::optional<std::string> getEnvironmentVariable(const char* variableName)
+    {
+#ifdef _MSC_VER
```

Comment:
I think _WIN32 would be more appropriate: as I understand it, `getenv_s` is a Windows API, not a VC++ API.

Reply:
Fixed.